lochness/mediaflux/__init__.py

```diff
@@ -163,6 +163,18 @@ def sync_module(Lochness: 'lochness.config',
                                     if 'out_dir' in prod else mf_local) / \
                                         subpath.parent
 
+                            # do not re-download already downloaded data
+                            prev_checksum_file = Path(mf_local).parent / \
+                                    f'.checksum_{Path(mf_local).name}'
+                            if prev_checksum_file.is_file():
```

Comment:
For each file, are you maintaining a separate checksum file?

Reply:
Yes, that's the idea.